### PR TITLE
Add stow and dependencies

### PR DESCRIPTION
**Pull request by 0xaf1f**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by 0xaf1f** [comment]
@conda-forge-admin, please ping team

As for the failing linter, when I run it locally, I just see

```
. has some lint:
  Selectors are suggested to take a ``<two spaces>#<one space>[<expression>]`` form. See lines [15]
```
which, for the stow package is just the build `number: 0` field...

**Comment by 0xaf1f** [comment]
not stale.

**Comment by jakirkham** [comment]
@conda-forge/bioconda-recipes ?

**Comment by 0xaf1f** [comment]
it's not related to biology/bionformatics. That's why I'm submitting it here instead of to bioconda.

Also the linter shows that it's clean when I ran it locally.

**Comment by jakirkham** [comment]
Yeah though they have more Perl experts 😉

File: recipes/perl-capture-tiny/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+package:
+  name: perl-capture-tiny
+  version: {{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN/Capture-Tiny-{{ version }}.tar.gz
+  sha256: 6c23113e87bad393308c90a207013e505f659274736638d8c79bac9c67cc3e19
+
+build:
+  noarch: generic
```
Comment (mbargull) [inline recipes/perl-capture-tiny/meta.yaml:12]:
```suggestion
```
We can't have `noarch: generic` Perl packages, yet (buthopefully soon-ish).
Building a `noarch: generic` package on Linux would make it installable but not runnable on other platforms, e.g., Windows. So for now, we have to have them in the platform-specific subdirs, not `noarch`.

**Review by mbargull — CHANGES_REQUESTED** [review]

File: recipes/perl-capture-tiny/meta.yaml (hunk 1 above)
Comment (0xaf1f) [reply recipes/perl-capture-tiny/meta.yaml]: Thanks for reviewing. Deleted this line, squashed, and force-pushed.

**Comment by mbargull** [comment]
@conda-forge-admin, please lint

**Comment by 0xaf1f** [comment]
I don't know what the deal is with that. This is what I get running the linter locally

```
staged-recipes/recipes/perl-capture-tiny$ conda smithy recipe-lint .
. is in fine form
staged-recipes/recipes/perl-capture-tiny$ cd ../perl-io-stringy
staged-recipes/recipes/perl-io-stringy$ conda smithy recipe-lint .
. is in fine form
staged-recipes/recipes/perl-io-stringy$ cd ../perl-test-output
staged-recipes/recipes/perl-test-output$ conda smithy recipe-lint .
. is in fine form
staged-recipes/recipes/perl-test-output$ cd ../stow
staged-recipes/recipes/stow$ conda smithy recipe-lint .
. is in fine form
```

File: recipes/perl-capture-tiny/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    - Capture::Tiny
+
+about:
+  home: https://github.com/dagolden/Capture-Tiny
+  license: Artistic-2.0
+  summary: 'Capture STDOUT and STDERR from Perl, XS or external programs'
+
+extra:
+  recipe-maintainers:
+    - 0xaf1f
```
Comment (isuruf) [inline recipes/perl-capture-tiny/meta.yaml:33]:
```suggestion
    - '0xaf1f'
```

Comment (0xaf1f) [reply recipes/perl-capture-tiny/meta.yaml]: done

Comment (0xaf1f) [reply recipes/perl-capture-tiny/meta.yaml]: yay now the linter service didn't fail

**Comment by 0xaf1f** [comment]
@conda-forge-admin, please ping team

The requested changes have all been applied. I think github is just not detecting that because I amended/force-pushed.

File: recipes/stow/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - perl
+    - perl-test-output
+    - perl-io-stringy
```
Comment (isuruf) [inline recipes/stow/meta.yaml:24]:
```suggestion
```

**Comment by 0xaf1f** [comment]
all requested changes have been applied. I think the only reason that @mbargull's requested changes are showing as outstanding is that github didn't notice that I addressed it since I had amended and force-pushed the original commit.

**Comment by 0xaf1f** [comment]
are we waiting for anything else? it's been over a year.

**Comment by 0xaf1f** [comment]
@mbargull @isuruf can we merge?

**Pull request merged**